source/contracts/extensions/MarketCreation.sol

```diff
@@ -9,7 +9,7 @@ import 'reporting/IReportingWindow.sol';
 
 contract MarketCreation {
     function createMarket(IUniverse _universe, uint256 _endTime, uint8 _numOutcomes, uint256 _feePerEthInWei, ICash _denominationToken, uint256 _numTicks, address _designatedReporterAddress) payable public returns (IMarket) {
-        IReportingWindow _reportingWindow = _universe.getReportingWindowByMarketEndTime(_endTime, (_designatedReporterAddress != 0));
+        IReportingWindow _reportingWindow = _universe.getReportingWindowByMarketEndTime(_endTime, true);
```

Comment:
This shouldn't have changed.  This was my bad, I think I accidentally left it in when I PRed into your branch but it shouldn't be here. `_designatedReporterAddress != 0` is the correct value for this.

Reply:
Yeah, I wasn't sure why that had changed and was meaning to ask you about it.  I'll change it back.